src/web/components/elements/FullReview.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { headline, textSans } from '@guardian/src-foundations/typography';
+import { space } from '@guardian/src-foundations';
+
+import { pillarPalette } from '@root/src/lib/pillars';
```

Comment:
We should be using `palette`. Never `pillarPalette`.

Reply:
as in `import { news } from '@guardian/src-foundations/palette';`?

Do we not want the text colour to be linked to the pillar?

Reply:
In DCR, we use `palette` for colours. As in [decidePalette](https://github.com/guardian/dotcom-rendering/blob/ea111975f28140ca6fabd9c357ce5428bb1c32bb/src/web/lib/decidePalette.ts#L720).

Because we have our _own_ `palette` we never use the Source one (except in that _one_ file). If you search DCR for `pillarPalette` (outside of amp) you will only find it being used in 3 files. Two of those files are not used anywhere and are dead, the other one is `decidePalette`

Reply:
I have removed `pillarPalette`, instead letting the global scope set the colour of `a` tags